src/data/bool/count.lean

```diff
@@ -4,6 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury Kudryashov
 -/
 import data.nat.parity
+import data.list.count
```

Comment:
```suggestion
```

Reply:
Before this PR, `data.nat.parity` imported `data.list.count`.

Reply:
The file compile without this import. So you should remove it from this file (and add it to other files if necessary).